src/pages/DelegationPage/components/DelegationProvidersList.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="delegation-providers-list">
```

Comment:
if it is a list and it looks like list it's better to use semantically matching html elements (`<ul>`, `<ol>`, `<li>`)